### PR TITLE
Add Progressbar hackerspace hexagon

### PR DESCRIPTION
**Pull request by yangwao**
IMG

### CODE REVIEW

**Comment by yangwao** [comment]
bump? :) 
Maybe I could help to maintain this repository?

File: meta/progressbar.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "name": "progressbar",
+  "author": "wao + zden",
+  "license": "BSD",
+  "vector": "http://hexb.in/vector/progressbar.svg",
+  "description": "Progressbar hackerspace Bratislava https://progressbar.sk",
+  "raster": "http://hexb.in/hexagons/progressbar.png",
+  "filename": "meta/progressbar.json",
+  "order_online_url": ""
```
Comment (Ruxton) [inline meta/progressbar.json]: Please remove order_online_url if you don't have a value for it.

**Review by Ruxton — DISMISSED** [review]

**Comment by yangwao** [comment]
@Ruxton fixed :)

**Comment by yangwao** [comment]
Tahnks @Ruxton ! 🤗
But I don't see it at http://hexb.in/ or needs to be regenerated manually? Should I change PR to help?

**Pull request merged**